terminal/bundles/org.eclipse.terminal.view.ui/src/org/eclipse/terminal/view/ui/internal/handler/NewTerminalViewHandler.java

```diff
@@ -24,10 +26,12 @@ public class NewTerminalViewHandler extends AbstractTriggerCommandHandler {
 
 	@Override
 	public Object execute(ExecutionEvent event) throws ExecutionException {
-		UIPlugin.getConsoleManager().showConsoleView(new TerminalViewId().next());
-
-		triggerCommand("org.eclipse.terminal.view.ui.command.launchToolbar", null); //$NON-NLS-1$
-
+		try {
+			UIPlugin.getConsoleManager().showConsoleView(new TerminalViewId().next());
+			triggerCommand("org.eclipse.terminal.view.ui.command.launchToolbar", null); //$NON-NLS-1$
+		} catch (CoreException e) {
+			ILog.get().log(e.getStatus());
```

Comment:
Should this maybe better be


```suggestion
			throw new ExecutionException(e);
```

Reply:
done